core/shared/src/main/scala/io/circe/Decoder.scala

```diff
@@ -1,11 +1,13 @@
 package io.circe
 
+import java.nio.ByteBuffer
+
 import cats.Monad
 import cats.data._
 import cats.std.list._
 import cats.syntax.cartesian._
 import cats.syntax.functor._
-import java.util.UUID
+import java.util.{ Base64, UUID }
```

Comment:
I think `DatatypeConverter` makes sense until we drop Java 7 support.

Any reason we don't want to drop it now? As I recall, 1.7 eol-ed on Apr 2015.